modPrecompiled/test/org/aion/precompiled/contracts/DummyRepo.java

```diff
@@ -172,6 +172,9 @@ public void flush() {}
     @Override
     public void flushTo(Repository repo, boolean clearStateAfterFlush) {}
 
+    @Override
+    public void flushCopiesTo(Repository repo, boolean clearStateAfterFlush) {}
```

Comment:
is this change supposed to be in this PR?

Reply:
Yes. In order to reduce the reliance of the FVM on the repository cache implementation, I had to extend the interface functionality: https://github.com/aionnetwork/aion_fastvm/pull/56/files

Reply:
The alternative would have been to add `modAionImpl` as a dependency, which I thought was going in the wrong direction.